src/dmd/dsymbolsem.d

```diff
@@ -4764,6 +4764,29 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         {
             auto s = (*cldec.members)[i];
             s.dsymbolSemantic(sc2);
+
+            if (cldec.classKind == ClassKind.cpp)
+            {
+                DtorDeclaration dtor = s.isDtorDeclaration();
```

Comment:
Can this variable be `const`?

Reply:
probably... is this another case where i'm expected to write code differently than all the surrounding code, and I'm just meant to know that? :P

Reply:
Yes :smiley:.